src/prerenderContent/generatePreloadDump.js

```diff
@@ -1,3 +1,13 @@
 export default function generatePreloadDump(preloadDump) {
-  return `window.preloadDump = ${JSON.stringify(preloadDump)};`;
+  return `window.preloadDump = ${stringLiteral(preloadDump)};`;
+}
+
+/**
+ * JSON may contain line separators, which are invalid in string literals in older browsers.
+ * See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#issue_with_plain_json.stringify_for_use_as_javascript
+ * */
+function stringLiteral(string) {
+  return JSON.stringify(string)
+    .replace(/\u2028/g, "\\u2028")
+    .replace(/\u2029/g, "\\u2029");
```

Comment:
How does this behave with newer browsers, that already implement https://github.com/tc39/proposal-json-superset ? Will a later `JSON.parse` of such a string still produce the same result?

Reply:
Yes, every \uHHHH (4 hex digits) is a valid backslash-UnicodeEscapeSequence according to https://262.ecma-international.org/5.1/#sec-7.8.4

Reply:
Also note, it is not about JSON.parse, but parsing source code.

Reply:
I know, that this is about parsing source code. But that parsed source code will eventually be put into a `JSON.parse` (by `Scrivito.preload`). So I want to make sure, that our bugfix does not introduce a new bug for engines, that already implement https://github.com/tc39/proposal-json-superset.

Reply:
```js
eval(`"${String.fromCodePoint(0x2028)}"`) === "\u2028"
// true
```